content/integrations/aws-secrets-manager.md

```diff
@@ -227,3 +227,135 @@ You need an [AWS access key and AWS secret key pair](https://docs.aws.amazon.com
    }
    ```
+
+## Using KES with a MinIO Server
+
+MinIO Server requires KES to set up server-side data encryption.
+
+See the [KES for MinIO instruction guide]({{< relref "/tutorials/kes-for-minio.md" >}}) for additional steps needed to use your new KES Server with a MinIO Server.
+
+
+## Configuration References
+
+The following section describes each of the Key Encryption Service (KES) configuration settings for using AWS Secrets Manager and AWS Key Management System as the root KMS for Server Side Encryption with KES.
+
+{{< admonition title="MinIO Server Requires Expanded Permissions" type="important" >}}
+Starting with [MinIO Server RELEASE.2023-02-17T17-52-43Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-02-17T17-52-43Z), MinIO requires expanded KES permissions for functionality. 
+The example configuration in this section contains all required permissions.
+{{< /admonition >}}
+
+
+{{< tabs "aws-config" >}}
+{{< tab "YAML Overview" >}}
+Fields with `${<STRING>}` use the environment variable matching the `<STRING>` value. 
+You can use this functionality to set credentials without writing them to the configuration file.
+
+The YAML assumes a minimal set of permissions for the MinIO deployment accessing KES. 
+As an alternative, you can omit the `policy.minio-server` section and instead set the `${MINIO_IDENTITY}` hash as the `${ROOT_IDENTITY}`.
+
+```yaml {.copy}
+address: 0.0.0.0:7373
+root: ${ROOT_IDENTITY}
+
+tls:
+  key: kes-server.key
+  cert: kes-server.cert
+
+api:
+  /v1/ready:
+    skip_auth: false
+    timeout:   15s
+
+policy:
+  minio-server:
+    allow:
+    - /v1/key/create/*
+    - /v1/key/generate/*
+    - /v1/key/decrypt/*
+    - /v1/key/bulk/decrypt
+    - /v1/key/list/*
+    - /v1/status
+    - /v1/metrics
+    - /v1/log/audit
+    - /v1/log/error
+    deny:
+    - /v1/key/generate/my-app-internal*
+    - /v1/key/decrypt/my-app-internal*
+    identities:
+    - ${MINIO_IDENTITY}
+
+    my-app:
+    allow:
+    - /v1/key/create/my-app*
+    - /v1/key/generate/my-app*
+    - /v1/key/decrypt/my-app*
+    deny:
+    - /v1/key/generate/my-app-internal*
+    - /v1/key/decrypt/my-app-internal*
+    identities:
+    - df7281ca3fed4ef7d06297eb7cb9d590a4edc863b4425f4762bb2afaebfd3258
+    - c0ecd5962eaf937422268b80a93dde4786dc9783fb2480ddea0f3e5fe471a731
+
+keys:
+  - name: "minio-encryption-key-alpha"
+  - name: "minio-encryption-key-baker"
+  - name: "minio-encryption-key-charlie"
+
+cache:
+  expiry:
+    any: 5m0s
+    unused: 20s
+    offline: 0s
+
+# The following log configuration only affects logging to console.
+log:
+
+  # Enable/Disable logging error events to STDERR. Valid values
+  # are "on" or "off". If not set the default is "on". If no error
+  # events should be logged to STDERR it has to be set explicitly
+  # to: "off".
```

Comment:
A little more direct alternative?
```suggestion
  # Enable/Disable logging error events to STDERR. 
  # Valid values are "on" or "off". Defaults to "on".
  # To prevent logging error events to STDERR, set to "off".
```

There's some verbose comments in this sample configuration, might be worth looking at others as well.

Reply:
This was just copy/paste. I will simplify throughout.

Reply:
Especially because I have the reference tab, so the comments are needed in the yaml.

Reply:
How about after that I make a PR to improve the source, then yours will match 😁